### PR TITLE
Fixed CPU consumption of `infoPanel`

### PR DESCRIPTION
**Pull request by introkun**
* added 100ms sleep between reading keystrokes

### CODE REVIEW

**Comment by introkun** [comment]
Fixes #110

**Comment by introkun** [comment]
image
After applying the fix (sleep 100ms)

File: src/infoPanel/infoPanel.c [hunk 1]
```diff
@@ -197,6 +197,7 @@ int main(int argc, char *argv[])
 				}
 			}
 		}
+		msleep(100);
```
Comment (Aemiii91) [inline src/infoPanel/infoPanel.c:200]: 100ms ruins the experience! Maximum 15ms, I recommend 4ms.

**Review by Aemiii91 — CHANGES_REQUESTED** [review]

File: src/infoPanel/infoPanel.c (hunk 1 above)
Comment (introkun) [reply src/infoPanel/infoPanel.c]: done

**Review by Aemiii91 — APPROVED** [review]

**Pull request merged**